run.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+# Use in the the functions: eval $invocation
+invocation='echo "Calling: ${FUNCNAME[0]}"'
```

Comment:
All checked-in shell scripts should have `set -e` as their first line, so errors are handled correctly.